### ISSUE TITLE
error with java 9; is it supported?

### ISSUE THREAD

**Issue by ValeryVS** [allure-framework/allure2#638]
[//]: # (
. Note: for support questions, please use Stackoverflow or Gitter**. 
. This repository's issues are reserved for feature requests and bug reports.
.
. In case of any problems with Allure Jenkins plugin** please use the following repository 
. to create an issue: https://github.com/jenkinsci/allure-plugin/issues
.
. Make sure you have a clear name for your issue. The name should start with a capital 
. letter and no dot is required in the end of the sentence. An example of good issue names:
.
. - The report is broken in IE11
. - Add an ability to disable default plugins
. - Support emoji in test descriptions
)

#### I'm submitting a ... 
  - [x] bug report
  - [ ] feature request
  - [ ] support request => Please do not submit support request here, see note at the top of this template.

#### What is the current behavior?

With java 9 I have an error

#### What is the expected behavior?

Some info about supported version of java.

#### Please tell us about your environment:

node 6.9.5
allure-commandline@2.3.5

#### Other information 

```
Exception in thread "main" java.lang.NoClassDefFoundError: javax/xml/bind/JAXBException
	at io.qameta.allure.allure1.Allure1Plugin.xmlFiles(Allure1Plugin.java:393)
	at io.qameta.allure.allure1.Allure1Plugin.getStreamOfAllure1Results(Allure1Plugin.java:388)
	at io.qameta.allure.allure1.Allure1Plugin.readResults(Allure1Plugin.java:103)
	at io.qameta.allure.ReportGenerator.lambda$readResults$0(ReportGenerator.java:32)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1378)
	at io.qameta.allure.ReportGenerator.readResults(ReportGenerator.java:32)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195)
	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:177)
	at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1492)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
	at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:913)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:511)
	at io.qameta.allure.ReportGenerator.generate(ReportGenerator.java:54)
	at io.qameta.allure.ReportGenerator.generate(ReportGenerator.java:43)
	at io.qameta.allure.Commands.generate(Commands.java:88)
	at io.qameta.allure.CommandLine.run(CommandLine.java:129)
	at java.base/java.util.Optional.orElseGet(Optional.java:361)
	at io.qameta.allure.CommandLine.main(CommandLine.java:72)
Caused by: java.lang.ClassNotFoundException: javax.xml.bind.JAXBException
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:582)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:185)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:496)
	... 20 more
```

[//]: # (
. e.g. detailed explanation, stacktraces, related issues, suggestions 
. how to fix, links for us to have more context, eg. Stackoverflow, Gitter etc
)

**Comment by baev** [comment]
@ValeryVS fixed in 2.4.1

**Comment by egorvas** [comment]
@baev does it should work for a allure-maven plugin? I have the same error with java 9.
